src/backend/wsl.ts

```diff
@@ -648,13 +648,13 @@ export default class WSLBackend extends events.EventEmitter implements VMBackend
       const defaultConfig = { credsStore: 'rancher-desktop' };
       let existingConfig: Record<string, any>;
 
-      await this.writeFile('/etc/init.d/credhelper-vtunnel-peer', SERVICE_CREDHELPER_VTUNNEL_PEER, { permissions: 0o755 });
-      await this.writeConf('credhelper-vtunnel-peer', {
+      await this.writeFile('/etc/init.d/vtunnel-peer', SERVICE_VTUNNEL_PEER, { permissions: 0o755 });
```

Comment:
Sounds like for dealing with upgrades you'll need to delete existing instances of `/etc/init.d/credhelper-vtunnel-peer` and `/etc/conf.d/credhelper-vtunnel-peer`